### PR TITLE
Update README.md

### LINKED ISSUE #644: Improving the readme file
I would like to work on improving the readme file of this project making it engaging for the users and make it more expressive.

### PR DESCRIPTION
**Pull request by harshsingh32**
## Fixes Issue
This pr is under issue #644.

closes #644

### CODE REVIEW

**Review by CBID2 — APPROVED** [review]
Well done @harshsingh32! 😊 LGTM! 😊

**Review by CBID2 — APPROVED** [review]
Great job @harshsingh32! :) LGTM! :)

File: README.md [hunk 1]
```diff
@@ -142,6 +142,8 @@ and the website will be built in one single click. No extra setup is required.
 
 ## Our Contributors ✨
 
+We would like to express our sincere gratitude to all the contributors who have helped make LinksHub an invaluable resource for developers. Without your contributions, this project wouldn't have been possible. Thank you for your time, effort, and dedication.
```
Comment (rupali-codes) [inline README.md:145]:
```suggestion
We would like to express our sincere gratitude to all the contributors who have helped make LinksHub an invaluable resource for developers. Without your contributions, this project wouldn't have been possible. Thank you so much for your time, efforts, and dedication.
```

**Review by rupali-codes — APPROVED** [review]
looks good

**Pull request merged**